### PR TITLE
More styles

### PR DESCRIPTION
**Pull request by Dignifiedquire**
* Toggle the sidebar whenever you want (shown by default on desktop, hidden on smaller screen sizes)
* Add blockquote styling
* Minor tweaks

### CODE REVIEW

File: assets/js/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  var dur = duration
+  if (immediate) {
+    dur = 0
+  }
+  sidebar.animate({
+    '-webkit-flex-basis': width,
+    '-moz-flex-basis': width,
+    '-ms-flex-basis': width,
+    'flex-basis': width
+  }, 0, function () {
```
Comment (milmazz) [inline assets/js/sidebar.js]: I think that at this point you should reference the variable `dur` to indicate the duration and not a fixed value to perform the animation, right?

File: assets/js/sidebar.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    openSidebar()
+  }
+}
+
+function init () {
+  $('.sidebar-toggle').click(function () {
+    toggleSidebar()
+  })
+
+  if ($(window).width() > 768) {
```
Comment (milmazz) [inline assets/js/sidebar.js]: IMO, this kind of detection is more related with styling stuff... So, why not to use a *css media query* in this case?

**Comment by milmazz** [comment]
@Dignifiedquire I made a few comments in your code. Also, could you please fix the ESLint error. Thanks!

File: assets/js/sidebar.js (hunk 2 above)
Comment (Dignifiedquire) [inline assets/js/sidebar.js]: @milmazz not sure what you mean? I can't add a class using a `css media query`

**Comment by Dignifiedquire** [comment]
@milmazz fixed

**Pull request merged**